slides/02.qmd

```diff
@@ -5,57 +5,32 @@ title: Names and values
 
 ## Learning objectives
 
-- To be able to understand distinction between an *object* and its *name*
-- With this knowledge, to be able write faster code using less memory
+- To distinction between an *object* and its *name*
+- To identify when data are _physically_ copied versus modified
 - To better understand R's functional programming tools
```

Comment:
```suggestion
- Distinguish between an *object* and its *name*.
- Identify when data are *copied* versus *modified*.
- Trace the memory used by R.
```
I like to make the LOs more like sentences. I know the existing slides had "To...", so this wasn't your fault 😊

Data aren't really "physically" copied, so I changed the stress to "copied" vs "modified".

This chapter briefly mentions how this stuff is useful for functional programming, but we aren't really learning about those tools here, so I removed that LO, and added a more-specific one about what we're doing at the end of this chapter.